config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show, :new, :create]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create, :destroy] # Add comments resource
```

Comment:
- [ ] Kindly note that without a new route under comments we can't create a comment because a new action in the comments controller will not be called to render the form. `resourses : comments , only: [:new, :create, :show, :destroy ]`